doozerlib/olm/bundle.py

```diff
@@ -24,6 +24,25 @@ class OLMBundle(object):
     def __init__(self, runtime):
         self.runtime = runtime
 
+    def find_bundle_for(self, operator_nvr):
+        """Check if a bundle already exists for a given `operator_nvr`.
+
+        :param string operator_nvr: Operator NVR (format: my-operator-v4.2.30-202004200449)
+        :return string: NVR of latest found bundle build, or None.
+        """
+        self.operator_nvr = operator_nvr
+        self.get_operator_buildinfo()
+
+        builds = brew.get_tagged_builds(tags=[self.target],
+                                        build_type='image',
+                                        event=None,
+                                        session=self.brew_session)[0]
+        vr = self.operator_nvr.replace(self.operator_brew_component, '')[1:].replace('-', '.', 1)
+        bundle_nvrs = [build['nvr'] for build in builds]
+        found = list(filter(lambda bundle_nvr: vr in bundle_nvr, bundle_nvrs))
+        found.sort(reverse=True)
+        return found[0] if found else None
```

Comment:
I really don't like string-sorting nvrs. it sets a bad example - numbers don't sort correctly as strings. even if that would actually work here (since it would be highly unlikely for us to see more than 10 bundle builds for any operator nvr).

but with a record-based approach this could be...
```suggestion
        found.sort(reverse=True, key=lambda build: int(build['release']))
        return found[0]['nvr'] if found else None
```